modules/mono/glue/GodotSharp/GodotPlugins/Main.cs

```diff
@@ -91,6 +93,7 @@ internal void Unload()
             PluginsCallbacks* pluginsCallbacks, ManagedCallbacks* managedCallbacks,
             IntPtr unmanagedCallbacks, int unmanagedCallbacksSize)
         {
+            var isNativeFuncionsReady = false;
```

Comment:
- Renamed the variable to make it more obvious that this refers to the _initialized_ state of the `NativeFuncs` class.
- We tend to avoid `var` for primitive types.

```suggestion
            bool isNativeFuncsInitialized = false;
```